services/graphql-server/src/graphql/definitions/email/schedule.js

```diff
@@ -88,6 +89,13 @@ input EmailScheduleSortInput {
   order: SortOrder = desc
 }
 
+input EmailScheduleByDayQueryInput {
+  newsletterId: ObjectID!
+  startingStartOf: Date!
+  beforeEndOf: Date!
+  timezone: String # tz database format, e.g. America/Chicago
```

Comment:
Include GQL docs here indicating the purpose of these input fields, and expected format
```gql
"input description"
input inputname {
  "The newsletter product id"
  newsletterId: ObjectID!
  # ...
}
  ```

Do the same for the query above.